website/docusaurus.config.js

```diff
@@ -0,0 +1,71 @@
+// @ts-check
+// Note: type annotations allow type checking and IDEs autocompletion
+
+const lightCodeTheme = require('prism-react-renderer/themes/github');
+const darkCodeTheme = require('prism-react-renderer/themes/dracula');
+
+/** @type {import('@docusaurus/types').Config} */
+const config = {
+    title: 'graph-glue',
+    tagline: 'Dinosaurs are cool',
```

Comment:
modify example to our use case

Reply:
removed with eec2ae8bfe3ad019e15608ad916551bc64714e6c  
wasn't used already, as we don't have a traditional landing page